docs/cmdline-opts/ssl-no-ticket.d

```diff
@@ -0,0 +1,10 @@
+Long: ssl-no-ticket
+Help: Disable curl's use of SSL session-ticket rusing(OpensslSSL)
```

Comment:
typo!

Reply:
Also, as there's a generic support `--no-` prefixes to options, it would make more sense to have the option called `--ssl-ticket` to enable, and `--no-ssl-ticket` to disable.

Reply:
typo has been fixed.  
I refer to the --ssl-no-revoke option. 
The current implementation is disabled session ticket, so I do not want to change too much logic.  
In addition, I refer to the implementation of OpenSSL's -no_ticket option, I just want to implement a session reuse debugging option  (https://github.com/curl/curl/issues/698)